From c694222310d2bf84e612958192299278bae291ae Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Wed, 10 Apr 2024 11:40:51 -0600
Subject: [PATCH] remove unused streamlit-toggle import

---
 streamlit_vertical_slider/__init__.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/streamlit_vertical_slider/__init__.py b/streamlit_vertical_slider/__init__.py
index e9c56c1d..c6882bbe 100644
--- a/streamlit_vertical_slider/__init__.py
+++ b/streamlit_vertical_slider/__init__.py
@@ -3,7 +3,6 @@
 import streamlit.components.v1 as components
 import streamlit as st
 import altair as alt
-import streamlit_toggle as sts
 from typing import Literal
 
 
